src/util.ts

```diff
@@ -87,32 +120,66 @@ export const getNewErrors = (
   return result
 }
 
-export const getTotalErrorsCount = (errorMap: Map<string, ErrorInfo>): number =>
+export const getTotalErrorsCount = (
+  errorMap: Map<string, ErrorSummary>
+): number =>
   // NOTE: Previously, this was written with an array spread, but there was a bug
   // with microbundle that was incorrectly compiling that (see: https://github.com/TimMikeladze/tsc-baseline/issues/21).
   // Until that is resolved or the bundler is switched for this repo, this has been
   // rewritten with Array.from
   Array.from(errorMap.values()).reduce((sum, info) => sum + info.count, 0)
 
 export const toHumanReadableText = (
-  errorMap: Map<string, ErrorInfo>
+  errorSummaryMap: Map<string, ErrorSummary>,
+  specificErrorMap: Map<string, SpecificError>
 ): string => {
   let log = ''
 
-  for (const [key, error] of errorMap) {
+  for (const [key, error] of errorSummaryMap) {
+    const specificErrors = getSpecificErrorsMatchingSummary(
+      error,
+      specificErrorMap
+    )
+
     log += `File: ${error.file}\n`
     log += `Message: ${error.message}\n`
     log += `Code: ${error.code}\n`
+    log += `Hash: ${key}\n`
     log += `Count of new errors: ${error.count}\n`
-    log += `Hash: ${key}\n\n`
+    log += `${specificErrors.size} current error${
+      specificErrors.size === 1 ? '' : 's'
+    }:\n`
+
+    log += Array.from(specificErrors.values())
+      .map(
+        (specificError) =>
+          `${specificError.file}(${specificError.line},${specificError.column})`
+      )
+      .join('\n')
+
+    log += '\n\n'
   }
 
   return log.trim()
 }
 
+export const getSpecificErrorsMatchingSummary = (
+  errorSummary: ErrorSummary,
+  specificErrorMap: Map<string, SpecificError>
+): Map<string, SpecificError> => {
+  return new Map(
+    Array.from(specificErrorMap.entries()).filter(
```

Comment:
I'm wondering about possible concerns from this iterating through every specific error every time this function is called. My first thought was that I don't see the specific error hash being used so maybe it would be preferable to key the specific errors by file path first. Maybe like this:
```ts
type SpecificErrorsMap = {
  [filePath: string]: SpecificError[]
}
```

I think for many cases this won't matter, but if there is a major change to the number of TypeScript errors the current implementation could be slow. For example, if someone changes their tsconfig and now there are thousands of new errors or the root directory name is changed. This also makes me think of another suggestion I might make for this repo to limit the default number of output summaries for new errors to a bounded amount.

Reply:
I added #29 as a proposal for that last idea.

Reply:
Good point - thank you for thinking about performance there on the array filter.  I like the idea in #29 but since that SpecificError hash isn't used, I also think it's worthwhile to fix up the performance of this branch before getting into limiting the output results.  And your idea to key on the file path seems just right to me.

I'm out of time today, but I hope to update this branch to use the filePath key tomorrow.